### ISSUE TITLE
On era change page goes down for a minute

### ISSUE THREAD

**Issue by bhaskarSingh** [thevantageproject/PolkaAnalytics#40]
Right now when the era changes, new data is fetched and old data is deleted which leads to around 1 minute of downtime

**Comment by prastut** [comment]
@sahilnanda1995 🚩

Please get clarity and give clarity. I think our entire value prop dissolves if we don't have historical data to calculate rewards. At the moment, we are deleting all history it seems.

**Comment by bhaskarSingh** [comment]
Yes, for this we need to architect how we will be storing data in database for future use
